src/lib.rs

```diff
@@ -1163,4 +1177,37 @@ mod tests {
             Ok(())
         }
     }
+
+    #[cfg(feature = "arbitrary")]
+    mod arbitrary {
+        use crate::NonEmpty;
+        use arbitrary::{Arbitrary, Unstructured};
+
+        #[derive(Debug, Eq, PartialEq, arbitrary::Arbitrary)]
+        pub struct SimpleArbitrary(pub i32);
```

Comment:
You don't really need this newtype. 
You could use something trivial like `u8` or `i32` (meaning `NonEmpty<i32>` instead of `NonEmpty<SimpleArbitrary>`.

Reply:
(Same response as above, I do agree though)